pkg/apis/duck/v1/delivery_types.go

```diff
@@ -123,6 +128,17 @@ func (ds *DeliverySpec) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
+	if ds.Format != nil {
+		validFormats := map[string]bool{
+			"structured": true,
```

Comment:
@Cali0707 I was thinking now, since "structured" is not an event format but rather "JSON" is an event format, should we use `JSON`, so that if we would need to support "XML" or "protobuf" we could?

Reply:
That's a good point- let's use `JSON` instead then. @EraKin575 can you make the necessary changes?

Reply:
I will